packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -897,11 +899,26 @@ const NextStepsDialog = ({ onClose, userId, targetUserId, targetUserDisplayName,
               />)}
               
             </div>      
+            <TextField
+              variant="outlined"
+              label="You can share a calendly link for easier scheduling"
+              rows={2}
+              error={!calendlyLink.valid}
+              helperText={!calendlyLink.valid && "Please enter a valid calendly link"}
+              fullWidth
+              value={calendlyLink.url ?? ""}
+              margin="normal"
+              onChange={event => setCalendlyLink(validatedCalendlyUrl(event.target.value))}
+            />
+            { calendlyLink.valid && calendlyLink.url && <>
+              <h3 className={classes.dialogueFormatHeader}>A preview of what we'll show your partner</h3>
+              <CalendlyIFrame url={calendlyLink.url}></CalendlyIFrame>
```

Comment:
`<CalendlyIFrame url={calendlyLink.url}></CalendlyIFrame>` can be `<CalendlyIFrame url={calendlyLink.url} />`.